run.go

```diff
@@ -16,20 +16,29 @@ package terramate
 
 import (
 	"fmt"
+	"os"
 	"os/exec"
+	"path/filepath"
 
 	"github.com/madlambda/spells/errutil"
+	"github.com/mineiros-io/terramate/project"
 	"github.com/mineiros-io/terramate/stack"
 )
 
 const ErrRunCycleDetected errutil.Error = "cycle detected in run order"
 
-func Run(stacks []stack.S, cmdSpec *exec.Cmd) error {
+func Run(root string, stacks []stack.S, cmdSpec *exec.Cmd) error {
+	wd, err := os.Getwd()
```

Comment:
Shouldn't Run use the wd defined [here](https://github.com/mineiros-io/terramate/pull/104/files#diff-3a3a4d5e49230d35db58ed019f072d3b3fb7d818e9c255c2208d68bc4bc08578R188) ?